sdk/rust/oak_tests/src/lib.rs

```diff
@@ -777,9 +867,9 @@ pub type NodeMain = fn(u64) -> i32;
 /// at start-of-day (e.g. based on a build configuration), or by calling
 /// oak_tests::init_logging() first, and ignoring the subsequent failure from
 /// oak_log::init().
-pub fn start<S: ::std::hash::BuildHasher>(
+pub fn start(
     config: proto::manager::ApplicationConfiguration,
-    entrypoints: HashMap<String, NodeMain, S>,
+    entrypoints: HashMap<String, NodeMain, RandomState>,
```

Comment:
"

Reply:
Now that the `entrypoints` map is being stored, this function can no longer be a generic function that takes any kind of `HashMap<String, NodeMain, S>` &ndash; it has to specifically take a `HashMap<String, NodeMain, RandomState>` (as that is what's stored in the runtime).

However, leaving off the default `RandomState` in the parameter type gives "warning: parameter of type `HashMap` should be generalized over different hashers", so it seems it needs to be explicit.

Reply:
Alright, but here you are not generalising it, in fact you are specialising it? I guess the way to generalise it would be to introduce a type parameter `<S: BuildHasher>`, which I expect in most (all?) cases will be automatically inferred. Anyways up to you really, in practice I don't think we care about different hashers.

Reply:
Yes, it's getting explicitly specialized because the generic version can't be stored in a `struct`.  AIUI a fully-specialized version has to be stored, which means we need to take the same fully-specialized version of the type as the parameter here (or else deal with conversions between `HashMap<String, NodeMain, S>` and `HashMap<String, NodeMain, RandomState>` for arbitrary `S`).